superset/views.py

```diff
@@ -1639,31 +1639,62 @@ def tables(self, db_id, schema):
 
     @api
     @has_access_api
+    @expose("/copy_dash/<dashboard_id>/", methods=['GET', 'POST'])
```

Comment:
although i guess our save method also uses shorthand with `save_dash`, so nbd. can leave as is.